### PR TITLE
Committing period changes to strip out period in the API.

### PR DESCRIPTION
**Pull request by jonlipp**

### CODE REVIEW

File: src/template/agentPagesAgentBrief.php [hunk 1]
```diff
@@ -92,6 +94,8 @@
 
 			</ul>
 
+			</div>
```
Comment (TheNewCoke) [inline src/template/agentPagesAgentBrief.php]: @jonlipp We'll need this end `div` tag moved back so `.wolfnet_aoLinks` and `.wolfnet_aoContact` are siblings again.

**Comment by TheNewCoke** [comment]
@jonlipp @asbaumgart I'm guessing this was closed because of the WP security issue interruption. After you make the change to the `.wolfnet_aoContact` div tag, you can submit a pull request to branch 1.14.9-rc.

**Comment by asbaumgart** [comment]
Yeah (re: WP security issue interruption). Probably want to remove the var_dump in src/AgentPagesHandler.php as well.

**Comment by jonlipp** [comment]
John, do you want me to make that change?  Is it just adding a closing <div> tag after the closing <ul> tag around line 97?  I am not sure why I didn't leave that there when updating the code, but I'll fix it if need be.  Please let me know either way.  I noticed another bug I introduced in there, so I'll fix that too.

**Pull request closed without merging**